### ISSUE TITLE
Auto-Search not working on Nemo v5.2.4 / Linux Mint v20.1

### ISSUE THREAD

**Issue by DrZorba** [linuxmint/nemo#3351]
### Distribution

Mint 20.1

### Package version

5.2.4

### Frequency

Always

### Bug description

Auto-Search isn't working on Nemo v5.2.4 on Linux Mint v20.1 

### Steps to reproduce

1. Open Nemo's "search" window by clicking on hand lens icon.

2. Begin typing in search window.

3. Search function does not initiate regardless how many characters are typed.
        Search will only initiate on hitting <Enter> key in the search window.

### Expected behavior

Nemo should begin searching on entry of the third character in the search string.

### Additional information

I corrected my problem (at least temporarily) by using Synaptic manager
        to roll back to v4.4.2, in which Auto-Search does work as intended.
        I have a second (older) install of Mint 20.1 w/Nemo v4.8.6, and Auto-search works perfectly
        in 4.8.6.  However, that version isn't available in Synaptic but is available in the Mint repository.
        http://packages.linuxmint.com/search.php?release=ulyssa&section=any&keyword=nemo
        I used the older package in Synaptic because it was simpler to lock the version
        than the manually-installed .deb package.

**Comment by Jeremy7701** [comment]
No, its always searched when you press the enter key, regardless of how many characters are entered, even if its only one character.

**Comment by DrZorba** [comment]
I have this very day seen with my own two eyeballs two different versions (4.8.6 & 4.4.2) on two different computers both of which display the auto-search behavior whenever three or more characters are entered.

And before I posted I sifted back through a couple of years of old gigs and found multiple references to this "auto-search" feature.

**Comment by claudiux** [comment]
Click once on a file or folder. Without using the "search" tools, type at least 1 character.

**Comment by Jeremy7701** [comment]
> Click once on a file or folder. Without using the "search" tools, type at least 1 character.

Not using the search feature - although it's available with CTRL + F...

It doesn't do any searching and I don't recall any function ever doing anything unless a control key (including ENTER) or a mouse operation was performed. So search is working as expected.

This is using Nemo 6.0.2 in LMDE6.
